bin/v-add-sys-ssh-jail

```diff
@@ -29,13 +29,28 @@ if [ ! -x /sbin/jk_init ]; then
 	exit
 fi
 
+# Checking if bubblewrap is installed
+if [ ! -x /bin/bwrap ]; then
+	exit
+fi
+
 # Perform verification if read-only mode is enabled
 check_hestia_demo_mode
 
 #----------------------------------------------------------#
 #                       Action                             #
 #----------------------------------------------------------#
 
+# Move jailbash to /usr/sbin
+if [ ! -x /usr/sbin/jailbash ]; then
+	cp -f $HESTIA_COMMON_DIR/bubblewrap/jailbash /usr/sbin/jailbash
```

Comment:
`2024-12-24T10:30:39.532773+00:00 dev sshd[257977]: User test not allowed because shell /usr/sbin/jailbash is not executable`

So we need to chmod +x /usr/sbin/jailbash